src/zombiereborn.cpp

```diff
@@ -857,7 +859,10 @@ void ZR_StartInitialCountdown()
 		if (g_iInfectionCountDown <= 60)
 		{
 			char message[256];
-			V_snprintf(message, sizeof(message), "First infection in \7%i %s\1!", g_iInfectionCountDown, g_iInfectionCountDown == 1 ? "second" : "seconds");
+			if (g_iInfectSpawnType != EZRSpawnType::RESPAWN)
+				V_snprintf(message, sizeof(message), "Classic spawns enabled! Zombies will be spawning between humans!\nFirst infection in \7%i %s\1!", g_iInfectionCountDown, g_iInfectionCountDown == 1 ? "second" : "seconds");
+			else
+				V_snprintf(message, sizeof(message), "First infection in \7%i %s\1!", g_iInfectionCountDown, g_iInfectionCountDown == 1 ? "second" : "seconds");
```

Comment:
You need to split this up, because this same buffer is being used for both hud msgs and chat msgs. As a result it spams the chat a lot more than once too.

Was this even tested? Because the chat formatting is also garbled due to newline in chat messages being interpreted as gray colour.
![image](https://github.com/Source2ZE/CS2Fixes/assets/6075172/78910efb-84b9-4e28-b421-59f1d16a9199)

Reply:
I couldn't test it because I don't even have a test server. I'll fix it later afterwards.